bin/format-cpan-uploads

```diff
@@ -71,10 +71,11 @@ for my $release (sort @lines)
       sprintf('%s %s', $module{mcpan_link}, $module{description}),
       sprintf('%s can %s', $module{mcpan_link}, $module{description}),
     );
-    $module{description} = $descriptions[ int rand @descriptions ];
+    #$module{description} = $descriptions[ int rand @descriptions ];
```

Comment:
I think rather than commenting out code, we can delete it. It's easy enough to revert changes since they're tracked by `git` anyway.